client/src/components/Sidebar/Chat.js

```diff
@@ -18,57 +18,51 @@ const styles = {
       cursor: "grab",
     },
   },
-};
+}));
 
-class Chat extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {noUnRead: false}
-  };
+const Chat = ({ conversation }) => {
+  const dispatch = useDispatch();
+  const classes = useStyles();
+  const [noUnread, setNoUnread] = useState(false);
+  const { otherUser, messages } = conversation;
+  const unReads = messages.filter(
```

Comment:
I trie for so long to get Sequelize to return a count, but just couldn't get it to work, so that's why I ended up with this front-end solution. Will keep trying though!